plugin.py

```diff
@@ -30,7 +30,11 @@ def plugin_info(self):
 
     def process(self, trace, data_context):
         with trace.open(data_type='text', mode='wb') as writer:
-            kaitai_utils.write_to_json(trace.open(), writer, kaitai_utils.get_kaitai_class())
+            kaitaiclass = kaitai_utils.get_kaitai_class()
+            with trace.open() as data:
```

Comment:
do you really need 2 seperate with statements?

Reply:
it might be possible to say `data.seek(0)`.

Reply:
is the with really a problem?

Reply:
you can perhaps simplify a little:
```python
with trace.open(data_type='text', mode='wb') as writer, trace.open() as data:
```

Reply:
done